clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -768,21 +768,13 @@ getTripleBasedSPIRVTransOpts(const ArgList &Args,
 }
 
 // Run LLVM to SPIR-V translation.
-static Expected<StringRef> runLLVMToSPIRVTranslation(StringRef InputTable,
+static Expected<StringRef> runLLVMToSPIRVTranslation(StringRef File,
```

Comment:
So far, this function and runAOTCompile function have been reading in from the SYCL table file and writing back to it. They have been modified to now work on individual inputs. The reading/writing of SYCL table file have been moved to a higher level of call hierarchy.

Reply:
It makes sense to extend the code comments before the function:

"Converts `File` file from LLVM bitcode to SPIR-V format using `llvm-spirv` tool. `Args` argument provides additional arguments for `llvm-spirv` tool."

Similar descriptions will be useful for all modified methods below.

Reply:
Will add the comments. Thanks.

Reply:
I have made a full parse of the file and added detailed comments for all SYCL specific functions.

Thanks